pkg/ibmvpc/ibmDrawioGenerator.go

```diff
@@ -10,6 +10,7 @@ func (v *VPC) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeN
 	return drawio.NewVpcTreeNode(gen.Cloud(), v.Name())
 }
 
+func (z *Zone) IsExternal() bool { return false }
```

Comment:
this should be where the other methods of `Zone` are defined

Reply:
IsExternal() and  GenerateDrawioTreeNode() are the methods in DrawioResourceIntf.

 since we decided to separate all GenerateDrawioTreeNode() from their structs, and implement them together in one file. I thought put IsExternal() in the same file
 what do you think?

Reply:
I think `IsExternal()` is not as related to `drawio` as the  `GenerateDrawioTreeNode` method.. Initially I thought it would better fit the `VPCResourceIntf` , but realized it is also required for grouped elements, and thus has to be in `DrawioResourceIntf`.  [though we can add this method to both `VPCResourceIntf` and `EndpointElem` interfaces instead?]

So I thought that at least this method belongs to where all the other "regular" methods are defined for each type. (it's not as unique as the special drawio functionality, and can be used for other cases).

Reply:
since the drawio code handle objects of DrawioResourceIntf, If I will move IsExternal() to VPCResourceIntf and EndpointElem, I wont be able to use it.
I will moving the methods to the files as you suggested, but keep it at DrawioResourceIntf.

Reply:
done